addons/mrp/report/mrp_production_templates.xml

```diff
@@ -139,12 +139,12 @@
     <t t-call="web.basic_layout">
         <t t-set='nRows' t-value='12'/>
         <t t-set='nCols' t-value='4'/>
-        <t t-set="uom_categ_unit" t-value="env.ref('uom.product_uom_categ_unit')"/>
+        <t t-set="uom_unit" t-value="env.ref('uom.product_uom_unit')"/>
         <div class="oe_structure"></div>
         <t t-set="move_lines" t-value="docs.move_finished_ids.move_line_ids.filtered(lambda ml: ml.move_id.production_id.state == 'done' and ml.state == 'done' and ml.quantity)"/>
         <t t-set="move_line_qtys" t-value="[]"/>
         <t t-foreach="move_lines" t-as="move_line">
-            <t t-if="move_line.product_uom_id.category_id == uom_categ_unit">
+            <t t-if="uom_unit.parent_path in move_line.product_id.uom_id.parent_path">
```

Comment:
`uom.uom` has no `parent_path`? :thinking:

Reply:
I added it: https://github.com/odoo/odoo/pull/184131/files#diff-7aa42e1663219ce82e34c884679a1f295e1fe26afd8daa7cb4d52d8663e79b83R13-R14, so that it's available to use if we want to do parent/child checks.

Reply:
Indeed, didn't reach that point yet. :+1: